File: Projects/NNN/nnn.js
convert = () => {
  let input = document.getElementById('nnn-input').value;
  let result = nullify(input);
  document.getElementById('nnn-result').textContent = result;
}

nullify = (input) => {
  let resultMid = [];
  for (let i = 0; i < input; i++) {
    resultMid.push(`{${resultMid.join(', ')}}`);
  }
  return `|{${resultMid.join(', ')}}|`;
}

File: Projects/NNN/nnn.py
from browser import document

def convert():
  input = document['nnn-input'].value
  result = nullify(input)
  document['nnn-result'] <= result

def nullify(input):
  intput = int(input)
  breakMe = document['breakMe'].value == 'on'
  if intput > 21 and not breakMe:
    return 'Specified number too high'
  else:
    resultMid = []
    for i in range(intput):
      resultMid.append(', '.join(resultMid))
    return f'|{resultMid}|'

def bindInit():
  document['nnn-input'].bind('onKeyUp', convert())
  print('convert() bound')

bindInit()
